docs/en/operation/Troubleshooting.md

```diff
@@ -269,6 +269,70 @@ are
 - `ADVANCED`: samples and tracks recent accesses, higher overhead
 - `PARANOID`: tracks for leaks on every resource allocation, highest overhead. 
 
+## Master Internal Monitoring
+
+Alluxio master periodically checks its resource usage, including CPU and memory usage, and several internal data structures 
+that are performance critical. This interval is configured by `alluxio.master.throttle.heartbeat.interval` (defaults to 3 seconds).
+On every sampling point in time (PIT), Alluxio master takes a snapshot of its resource usage. And a certain number of PIT snapshots
+(`alluxio.master.throttle.observed.pit.number`, defaults to 3) are recorded, those will be used to generate an aggregated
+one in a sliding window. 
```

Comment:
A continuous number of PIT snapshots (number configured by `alluxio.master.throttle.observed.pit.number`, defaults to 3) will be saved and used to generate the aggregated resource usage.

This line better to include more information, e.g. what the aggregated resource usage used for?

Reply:
i feel either we go simple, do not talk too much details e.g. Master keeps taking snapshots of resource usage and monitoring the overall healthiness based on the snapshots.

Or we give enough details about what snapshot and aggregated are used for